### ISSUE TITLE
[BUG] Windows 11 + Alt+Tab don't work well together

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-windows#1610]
**Describe the bug**
On Windows 10, the product was completely accessible using just the keyboard. On Windows 11, focus tracking and Alt+Tab interfere to the point that the product is almost unusable

**To Reproduce**
1. Run on Windows 11
2. Open several apps so you have a nice set of apps in the Alt+Tab list
3. Start AIWin with focus tracking enabled
4. Using Alt+Tab, try to shift the focus to an application that is several positions away in the Alt+Tab list

**Expected behavior**
You should be able to get to the app without any problems. This is the case on Windows 10

**Actual behavior**
The task switcher window gets focus, which inserts AIWin as the first window in the task switcher--it's almost impossible to get to the application in question. The problem goes away if you disable focus tracking, but that's the best way for keyboard users to interact with the tool.

**Screenshots or .GIF**

**Desktop (please complete the following information):**
 - OS: Windows 11 22H2
 - Accessibility Insights for Windows Version: 1.1.2312.1

**Additional context**
Priority requested - High

**Comment by codeofdusk** [comment]
This was filed as [Task 42637722](https://microsoft.visualstudio.com/OS/_workitems/edit/42637722) (internal access required to view). Also voting for high priority as this makes the app extremely hard to use for me!

**Comment by DaveTryon** [comment]
This should remain open until we release a version of AIWin using Axe.Windows 2.1.3

**Comment by DaveTryon** [comment]
This is fixed in https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.2360.01
